Contribution_1/Polygon3D.jl

```diff
@@ -0,0 +1,67 @@
+# This script creates 3D polygons from "x" and "y" coordinates, and allows you to define a depth range for the polygon.
+# It assigns phase and temperature values to the grid within the polygon.
+
+using GeophysicalModelGenerator     
```

Comment:
if you want this routine to be accessible in the GMG package, this file needs two be loaded in one of the other files with `include("Polygon3D.jl")`

Reply:
I am going to review this.